### ISSUE TITLE
Integrate write_eeg_data into scripts/data/write_annotate.py

### ISSUE THREAD

**Issue by F-said** [NDCLab/baseEEG#170]
**Background**
Integrate function into packages. this will then require a refactor of `run.py` but currently blocked by outerLoop. Will make new issue once blocker complete 

**Defined as Done**
Copy write_eeg_data into scripts/data/write_annotate.py and delete former file.
